src/Microsoft.Identity.Client/Platforms/winrt/DispatcherTaskExtensions.cs

```diff
@@ -37,18 +37,21 @@ internal static class DispatcherTaskExtensions
             Func<Task<T>> func, CoreDispatcherPriority priority = CoreDispatcherPriority.High)
```

Comment:
>High [](start = 89, length = 4)

does this have to be high?

Reply:
@henrik-me I do not know. I have not seen this file before. There is also a note on the documentation that says "Do not use this priority level in your app. It is reserved for system events. Using this priority can lead to the starvation of other messages, including system events."
(https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.coredispatcherpriority)

@jmprieur any ideas as to why we use CoreDispatcherPriority.High?